app/classification.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const teacherArray = ['taught','coached', 'trained', 'educated'];
+const studentArray = ['learned from'];
+const wroteArray = ['wrote','created', 'composed'];
+const playedArray = ['played','performed'];
+
+const dict = {teacher:teacherArray, student:studentArray, wrote:wroteArray, played:playedArray};
+
+module.exports.findRelationshipClass = function(word) {
+  for(let key in dict) {
+    let list = dict[key];
+    for(let w in list) {
+      if(list[w] === word) {
+        return key;
+      }
+    }
+  }
+  return null;
```

Comment:
I belive we are stored the classified relations separatly, so that should not be a concern.

What Im currently doing is using wordnet to gather the synonyms and use those instead of predefined them our self. This wont be perfect considering the taught to/by, but if we were to purely use wordnet or any other library we have looked at, they do not seem to coop well with these details anyway..